chapters.md

```diff
@@ -189,6 +189,28 @@ Chapter 2                | false              | no
 ++Nested Chapter 2.2.1   | true               | no
 ++Nested Chapter 2.2.2   | false              | no
 
+### ChapterSegmentUID
+
+
+The `ChapterSegmentUID` is a binary value with a size of 128 bits and the base element to set up a `Linked Chapter` in 2 variations. For both variations the following 3 conditions MUST be met.
+
+ 1. The `EditionFlagOrdered Flag` is set to true
+ 2. The `ChapterSegmentUID` is not the own `SegmentUID`
+ 3. The linked Matroska file/Segment is in the same folder
```

Comment:
I think "folder" it too specific. You could have a media library that keeps track of segment UIDs and tell you where to find them. That should be explained in the segment linking section. And this list item should mention that the UID must be a Segment UID that matches the conditions for segment linking.